### DISCUSSION TITLE
Pull request badge is missing

### DISCUSSION THREAD

**Discussion by sovetski** [community/community#55557] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Is it normal that I don't have the pull request badge on my profile? I have a pull request which was merged (on a repository which is not mine, I just contributed with fork), but I don't have the badge. Is there any cron which runs every 2/3 months to create badges, maybe?

**Comment by RevertIT** [comment]
You need 2 pull requests merged or more.

**Comment by YasinDehfuli** [comment, marked as answer]
### You can get Pull Shark Achievement by sumbit two accepted pull requests.
#### you can use this course to learn, [how to get Pull Shark](https://github.com/4xmen/Get-Github-Achievements/blob/main/assets/steps/README.pull-shark.md) Achievement Step-by-Step by image.

#### - hope its helpful for you & you get Pull Shark badge as soon as possible

**Comment by sovetski** [comment]
Thanks @RevertIT @YasinDehfuli !

**Reply by YasinDehfuli** [reply]
Your Welcome ❤️

**Comment by RajeevRanjan5642** [comment]


**Reply by YasinDehfuli** [reply]
Hi dear @sovetski I Think you have two merged in a private project. if your repo isn't private and you still don't get Pull Shark you can send an email to GitHub support. also you can use https://github.com/4xmen/Get-Github-Achievements/blob/main/assets/steps/README.pull-shark.md this project to better understand how to get pull shark achievement

**Reply by kuldeeprajput-dev** [reply]
hello bro, have you got the badge or still issue? because i am currently facing this issue, can you help me if your issue resolved ?

<img width="940" height="141" alt="image" src="https://github.com/user-attachments/assets/a49d3639-38a2-40cf-bbf4-06112fe3cd00" />



see my 2 pr merged 



<img width="763" height="669" alt="image" src="https://github.com/user-attachments/assets/0c6ba52f-8756-4a6c-8c41-6366336c9a49" />

see pull shark is not there
